nestjs/src/courses/interviews/interviews.service.ts

```diff
@@ -184,4 +193,118 @@ export class InterviewsService {
+      .addSelect('"stageInterview"."score" AS "interviewScore"')
+      .addSelect('"course"."name" AS "courseName"')
+      .addSelect('"course"."fullName" AS "courseFullName"')
+      .addSelect('"stageInterviewFeedback"."json" AS "interviewResultJson"')
+      .addSelect('"stageInterviewFeedback"."updatedDate" AS "interviewFeedbackDate"')
+      .addSelect('"stageInterviewFeedback"."version" AS "feedbackVersion"')
+      .addSelect('"userMentor"."firstName" AS "interviewerFirstName"')
+      .addSelect('"userMentor"."lastName" AS "interviewerLastName"')
+      .addSelect('"userMentor"."githubId" AS "interviewerGithubId"')
+      .addSelect('"courseTask"."maxScore" AS "maxScore"')
+      .leftJoin(Student, 'student', '"student"."id" = "stageInterview"."studentId"')
+      .leftJoin(User, 'user', '"user"."id" = "student"."userId"')
+      .leftJoin(Course, 'course', '"course"."id" = "stageInterview"."courseId"')
+      .leftJoin(
+        StageInterviewFeedback,
+        'stageInterviewFeedback',
+        '"stageInterview"."id" = "stageInterviewFeedback"."stageInterviewId"',
+      )
+      .leftJoin(CourseTask, 'courseTask', '"courseTask"."id" = "stageInterview"."courseTaskId"')
+      .leftJoin(Mentor, 'mentor', '"mentor"."id" = "stageInterview"."mentorId"')
+      .leftJoin(User, 'userMentor', '"userMentor"."id" = "mentor"."userId"')
+      .where('"user"."githubId" = :githubId', { githubId })
+      .andWhere('"stageInterview"."isCompleted" = true')
+      .orderBy('"course"."updatedDate"', 'ASC')
+      .getRawMany();
+
+    return data
+      .map((feedbackData: FeedbackData) => {
+        const {
+          feedbackVersion,
+          decision,
+          interviewFeedbackDate,
+          interviewerFirstName,
+          courseFullName,
+          courseName,
+          interviewerLastName,
+          interviewerGithubId,
+          isGoodCandidate,
+          interviewScore,
+          interviewResultJson,
+          maxScore,
+        } = feedbackData;
+        const feedbackTemplate = JSON.parse(interviewResultJson) as any;
+
+        const { score, feedback } = !feedbackVersion
+          ? InterviewsService.parseLegacyFeedback(feedbackTemplate)
+          : {
+              feedback: feedbackTemplate,
+              score: interviewScore ?? 0,
+            };
+
+        return {
+          version: feedbackVersion ?? 0,
+          date: interviewFeedbackDate,
+          decision,
+          isGoodCandidate,
+          courseName,
+          courseFullName,
+          feedback,
+          score,
+          interviewer: {
+            name:
+              this.userService.getFullName({ firstName: interviewerFirstName, lastName: interviewerLastName }) ||
+              interviewerGithubId,
+            githubId: interviewerGithubId,
+          },
+          maxScore,
+        };
+      })
+      .filter(Boolean);
+  }
+
+  /**
+   * @deprecated - should be removed once Artsiom A. makes migration of the legacy feedback format
```

Comment:
I think it's better to attach the issue number instead of writing that Artsiom will do it.
https://github.com/rolling-scopes/rsschool-app/issues/2281"